client/tests/e2e/permissions.js

```diff
@@ -28,8 +28,8 @@ test.serial("checking super user permissions", async t => {
   const $tooltip = await $("#role_ADVISOR_tooltip")
   t.regex(
     await $tooltip.getAttribute("title"),
-    /^Super users cannot create .*$/,
-    "Expected tooltip for super users"
+    /^Super-users cannot create .*$/,
```

Comment:
You should update the code as well if you change this (see `client/src/pages/people/Form.js`).